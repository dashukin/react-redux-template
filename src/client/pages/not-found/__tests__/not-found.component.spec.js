import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../not-found.component';

describe('Not found page', () => {
  it('should match NotFoundPage snapshot', () => {
    const wrapper = shallow(<NotFoundPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
