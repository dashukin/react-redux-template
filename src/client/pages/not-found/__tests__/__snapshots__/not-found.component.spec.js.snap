// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page should match NotFoundPage snapshot 1`] = `
<div>
  Not found page
</div>
`;
